docs/get-started/how-it-works.mdx

```diff
@@ -12,15 +12,23 @@ headers:
 
 In a nutshell, Earthstar helps you do three things:
 
-- Store and retrieve data
-- Synchronise that data
-- Prove the identity of who wrote that data
+- Store and retrieve data, by yourself or collaboratively in small groups
+- Synchronise that data across devices
+- Verify the integrity of the data against tampering
+
+This all works without a central server, but you can run server(s) to make it easier for devices to connect with each other.
 
 These things combined can be used to make rich collaborative applications, like chatrooms, games, or message boards.
 
+> **A note on vocabulary**
+>
+> We're still choosing names for the various pieces of Earthstar.  You may encounter older terminology, which we've highlighted in each section.
+
 ## Commons
 
-Commons symbolise groups interested in sharing data, like a Slack workspace or Discord server. They are made up of users who already know and trust each other.
+> Previously known as: *workspace*, *space*
+
+Commons are groups interested in sharing data, like a Slack workspace or Discord server.They are made up of users who already know and trust each other.
```

Comment:
Hm, this makes me think that the Discord / Slack metaphor might be misleading.

- A commons is a namespaced pool of data
- A commons doesn't have any notion of membership built into it, and accepts any document written to it
  - Therefore write access is strictly restricted to those with knowledge of the address
    - which is _most likely_ to be a group of people who know or trust each other
- Finally, all or part of that data **may be published publicly** (e.g. the web publishing use-case - blogs, read-only wikis, etc.)

Maybe it's better to describe a commons as a pool of data, where knowledge of the address grants you read/write access via a pocket? It's a bit of a mouthful but I hope you see what I'm getting at: I think it may be potentially confusing to describe a commons as a group of people.

Reply:
> Maybe it's better to describe a commons as a pool of data

How about: "A commons is like a shared folder."

Reply:
Yeah great hahah!